### PR TITLE
fix: add mingw32 to PATH

### PR DESCRIPTION
**Pull request by poppoerika**

### CODE REVIEW

File: .github/workflows/cd.yml [hunk 1]
```diff
@@ -168,6 +168,13 @@ jobs:
           asset_name: ${{ steps.create_release.outputs.asset_name }}
           asset_content_type: application/zip
 
+      - name: add mingw32 to path
+        shell: bash
+        run: |
+          echo "C:\msys64\mingw32\bin" >> $GITHUB_PATH
```
Comment (eaddingtonwhite) [inline .github/workflows/cd.yml:174]: this is kinda neat didnt know could do this and they have env var like this. 👍

Comment (poppoerika) [reply .github/workflows/cd.yml:174]: Thank you, Internet.

File: .github/workflows/cd.yml [hunk 2]
```diff
@@ -168,6 +168,13 @@ jobs:
...
+
+      - run: i686-w64-mingw32-gcc --version
```
Comment (eaddingtonwhite) [inline .github/workflows/cd.yml:176]: should we check exit code explicitly? or I guess that happens for us cause if this fails w/ non 0 exit code the action fails right?

**Review by eaddingtonwhite — APPROVED** [review]

File: .github/workflows/cd.yml (hunk 2 above)
Comment (poppoerika) [reply .github/workflows/cd.yml:176]: Yeah, the last action failed with exit code 1.

**Pull request merged**